### PR TITLE
fix: breacrumb.data to string,object, Add LOG level

### PR DESCRIPTION
**Pull request by HazAT**
## :loudspeaker: Type of change

- [X] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description
Add fix that breadcrumb.data is `<String, Object>` instead of `<String, String>`
Add `LOG` as log level.

## :bulb: Motivation and Context
Support for Hybrid SDKs

## :green_heart: How did you test it?
unit test.

## :pencil: Checklist

- [X] I reviewed submitted code
- [X] I added tests to verify changes
- [X] All tests passing

## :crystal_ball: Next steps
🚢

### CODE REVIEW

File: sentry-core/src/main/java/io/sentry/core/SentryLevel.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 /** the SentryLevel */
 public enum SentryLevel {
+  LOG,
```
Comment (bruno-garcia) [inline sentry-core/src/main/java/io/sentry/core/SentryLevel.java]: @HazAT This isn't in the docs eh?
Could you please clarify what this level is?

Oh, I assume it's some JS thing leaking in? Can we leave the value be mapped as unknown and travel through the serialization/deserialization instead?

Comment (marandaneto) [reply sentry-core/src/main/java/io/sentry/core/SentryLevel.java]: @bruno-garcia yes, this is JS leaking in.
this is a value and not an unknown key, as we have a strong typed `enum`, it won't work, unless we add a generic value to `SentryLevel` and assume it for all the unknown values.

Comment (bruno-garcia) [reply sentry-core/src/main/java/io/sentry/core/SentryLevel.java]: `LogLevel.Log` really makes no sense imho.
But sounds like the most pragmatic way to solve this though.

Can we have a javadoc on that explain where it comes from?

Comment (HazAT) [reply sentry-core/src/main/java/io/sentry/core/SentryLevel.java]: @bruno-garcia Added a javadoc

File: sentry-android-core/src/main/java/io/sentry/android/core/AndroidLogger.java [hunk 2]
```diff
@@ -39,39 +36,26 @@ public void log(SentryLevel level, String message, Throwable throwable) {
...
       case INFO:
         return Log.INFO;
       case WARNING:
         return Log.WARN;
       case FATAL:
         return Log.ASSERT;
-      case ERROR:
+      case LOG:
+      case DEBUG:
       default:
```
Comment (bruno-garcia) [inline sentry-android-core/src/main/java/io/sentry/android/core/AndroidLogger.java]: Sorry just spotted this and afaik default is error for issue events. 
But we should not be adding level by default so the server infers that

Comment (marandaneto) [reply sentry-android-core/src/main/java/io/sentry/android/core/AndroidLogger.java]: this is only used for calling the correct logging, not related to sentryEvent.level nor breadcrumb.level

Comment (bruno-garcia) [reply sentry-android-core/src/main/java/io/sentry/android/core/AndroidLogger.java]: My bad.

**Pull request merged**